File: kits/adjust/adjust-5/example/example-java/src/main/java/com/mparticle/kits/adjust/example/java/ExampleApplication.java
package com.mparticle.kits.adjust.example.java;

import android.app.Application;

import com.mparticle.MPEvent;
import com.mparticle.MParticle;
import com.mparticle.MParticleOptions;

public class ExampleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        MParticleOptions options =
                MParticleOptions.builder(this)
                        .credentials(
                                "REPLACE WITH YOUR MPARTICLE API KEY",
                                "REPLACE WITH YOUR MPARTICLE API SECRET")
                        .logLevel(MParticle.LogLevel.VERBOSE)
                        .build();
        MParticle.start(options);
        MParticle.getInstance()
                .logEvent(new MPEvent.Builder("foo", MParticle.EventType.Other).build());
    }
}

File: kits/braze/braze-40/example/example-kotlin/src/main/kotlin/com/mparticle/kits/braze/braze40/example/kotlin/ExampleApplication.kt
package com.mparticle.kits.braze.braze40.example.kotlin

import android.app.Application
import com.mparticle.MPEvent
import com.mparticle.MParticle
import com.mparticle.MParticleOptions

class ExampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        val options =
            MParticleOptions
                .builder(this)
                .credentials(
                    "REPLACE WITH YOUR MPARTICLE API KEY",
                    "REPLACE WITH YOUR MPARTICLE API SECRET",
                ).logLevel(MParticle.LogLevel.VERBOSE)
                .build()
        MParticle.start(options)
        MParticle.getInstance()?.logEvent(
            MPEvent.Builder("foo", MParticle.EventType.Other).build(),
        )
    }
}
